perf/rt_tests.py

```diff
@@ -32,14 +32,14 @@
 from avocado.utils import distro
 
 
-class rt_tests(Test):
+class RtTest(Test):
 
     def setUp(self):
         # Check for basic utilities
-        sm = SoftwareManager()
+        smm = SoftwareManager()
```

Comment:
i think should fine as pylint complains if variable length less than 3 

C: 39, 8: Invalid variable name "sm" (invalid-name)